Apromore-Core-Components/Apromore-Portal/pom.xml

```diff
@@ -69,6 +69,7 @@
                         <Import-Package>
                             org.eclipse.virgo.web.dm;resolution:=optional,
                             org.aopalliance.aop,                            
+                            org.apromore.security.exception,
```

Comment:
This is a bugfix not directly related to JDK 11 compatibility.  The missing package only causes trouble when an exception is thrown, which is why it wasn't noticed until I had to debug this PR.